cmd/scheduler.go

```diff
@@ -77,17 +77,102 @@ func getImqsHttpPort() int {
 
 func setDefaultVariables() {
 	imqsHttpPort := getImqsHttpPort()
-	config.Variables = make(map[string]string)
-	config.Variables["LOCATOR_SRC"] = "c:\\imqsvar\\imports"
-	config.Variables["LEGACY_LOCK_DIR"] = "c:\\imqsvar\\locks" // No longer needed, since we serialize all scheduled tasks. Should remove from imqstool.
-	config.Variables["JOB_SERVICE_URL"] = "http://localhost"
+	staticConfig.Variables = make(map[string]string)
```

Comment:
Don't call this static config when it actually holds values from both sets of config.